_events/2019-09-16-openhouse.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Open House \#2
+featured:
+location: Our house
+address: >-
+  114 N. Orchard St.
+  Madison, WI 53715
+---
+
+Join us for our second Open House of the fall 2019 semester! Starting at 6 Jimmy Johns and drinks will be available. Come any time and stay as long as you want. It’s guaranteed to be a packed house and a great chance to meet the members.
```

Comment:
Same issue with 'packed'

Reply:
Also, Capital One will be catering this event with Panera and Maddy will be giving a talk. Maybe something along the lines of: "Our very own alumna, Maddy Janick, a software engineer at Capital One will be giving a talk about professional life after college." A sentence about free panera would be good too